src/test/ui/span/issue-39018.stderr

```diff
@@ -46,10 +49,10 @@ LL |     let x = "Hello " + "World!".to_owned();
    |             |        `+` cannot be used to concatenate a `&str` with a `String`
    |             &str
    |
-help: `to_owned()` can be used to create an owned `String` from a string reference. String concatenation appends the string on the right to the string on the left and may require reallocation. This requires ownership of the string on the left
+help: call `.to_owned()` on the left and add a borrow on the right
```

Comment:
Perhaps I should remove the `.` from `.to_owned()` for consistency? Or alternatively add it to the other message?

Reply:
If you decide to keep the explicit reference to `to_owned` in the help messages, then I'd double check what other diagnostics do w/r/t referring to functions to call and try to be consistent with that.